wordpress-mobile/WordPress-iOS
# Dark mode: tab bar colors

Refs #12320

This fixes the colors of the tab bar icons with badges on them, as well as changes the colors to match the iOS system defaults:

![new tab colors](https://user-images.githubusercontent.com/517257/63646434-90b3ba00-c6c7-11e9-8a5a-0487ac64c393.png)

To test:
- check the tab bar icons in both light and dark mode
- check them out with and without badges (I fiddle with `WPTabBarController.m`)

Update release notes:
